app/views/help/cookie_settings.html.erb

```diff
@@ -1,15 +1,11 @@
 <% content_for :title, "Cookies on GOV.UK" %>
 <main id="content" role="main" class="group help-page cookie-settings govuk-grid-column-two-thirds">
-  <div class="cookie-settings__confirmation" data-cookie-confirmation="true">
-    <%= render "govuk_publishing_components/components/notice", {
-      title: t('cookies.confirmation_title'),
-      aria_live: true,
-    } do %>
-      <p><%= t('cookies.confirmation_message') %></p>
-      <a class="govuk-link cookie-settings__prev-page" href='#' data-module="track-click" data-track-category="cookieSettings" data-track-action="Back to previous page">
-        <%= t('cookies.confirmation_previous_referrer_link') %>
-      </a>
-    <% end %>
+    <div class="cookie-settings__confirmation" data-cookie-confirmation="true">
+      <%= render "govuk_publishing_components/components/success_alert", {
+        message: t('cookies.confirmation_title'),
+        description: "<p>#{t('cookies.confirmation_message')}</p>
+        <a class='govuk-link cookie-settings__prev-page' href='#' data-module='track-click' data-track-category='cookieSettings' data-track-action='Back to previous page'>#{t('cookies.confirmation_previous_referrer_link')}</a>".html_safe
```

Comment:
The component rendering in the gem doesn't include any sanitisation. This html includes dynamic content so I'd suggest adding a `sanitize` to this.